modules/tooling-support-parent/functional-tests/src/test/java/org/mule/runtime/module/tooling/ComponentValueProviderTestCase.java

```diff
@@ -48,24 +86,96 @@ public void actingParameterGroup() {
     final List<String> listValue = asList("one", "two", "three");
     ComponentElementDeclaration elementDeclaration =
         actingParameterGroupOPDeclaration(CONFIG_NAME, stringValue, intValue, listValue);
-    getResultAndValidate(session, elementDeclaration, PROVIDED_PARAMETER_NAME, "stringValue-0-one-two-three");
+    validateValuesSuccess(session, elementDeclaration, PROVIDED_PARAMETER_NAME, "stringValue-0-one-two-three");
   }
 
   @Test
-  public void complexActingParameter() {
+  public void actingParameterGroupWithDefaultValue() {
```

Comment:
We have agreed on not supporting default values when resolving metadata keys or valueProviders. Change the code and test to assert the error here.

Reply:
Added TODO in MTS